kittens/hints/main.go

```diff
@@ -257,6 +257,43 @@ func main(_ *cli.Command, o *Options, args []string) (rc int, err error) {
 		current_input = ""
 		current_text = ""
 	}
+	select_keys := make(map[int]string, len(index_map))
+	if o.CustomizeProcessing == "::import::kitty.choose_entry" {
+		for _, m := range index_map {
+			text := strings.TrimPrefix(m.Text, ": ")
+			if text == m.Text {
+				continue
+			}
+			key, _, ok := strings.Cut(text, " - ")
```

Comment:
This text parsing is quite ugly, to be fair. But not sure what to do - introducing structured input is not an option because of backwards compatibility. Maybe just add hotkey "tags" to every options line, e.g. `: option one <ctrl+1>`?